packages/sanity/src/core/studio/__telemetry__/studio.telemetry.ts

```diff
@@ -0,0 +1,12 @@
+import {defineEvent} from '@sanity/telemetry'
+
+interface StudioMountData {
+  studioVersion: string
+  plugins: {name: string}[]
+}
+
+export const StudioMount = defineEvent<StudioMountData>({
+  version: 1,
+  name: 'Studio Mounted',
+  description: 'The Studio was mounted on the page',
+})
```

Comment:
@bjoerge Would this trigger every time the Studio gets hard reloaded / opened in a new tab?

Reply:
@svirs I'm thinking that this would be similar to the identify event we have in the CLI? And since we already have the "Studio Entered" event set up server-side, this might be overlapping if it gets called every time (ref my comment above).

Reply:
> Would this trigger every time the Studio gets hard reloaded / opened in a new tab?

Yup!

Reply:
@svirs Need to figure out how to deal with this in addition to the current "Studio Entered" event. Maybe combine them if possible?

Reply:
I can change this to be

```ts
updateUserProperties({studioVersion: …, plugins: […]})
```